src/index.js

```diff
@@ -149,14 +152,29 @@ async function bundleMDX(
 
   const bundled = await esbuild.build(buildOptions)
 
-  const decoder = new StringDecoder('utf8')
+  if(bundled.outputFiles){
+    const decoder = new StringDecoder('utf8')
+
+    const code = decoder.write(Buffer.from(bundled.outputFiles[0].contents))
+
+    return {
+      code: `${code};return Component.default;`,
+      frontmatter,
+    }
+  }
+
+  if(buildOptions.outdir && buildOptions.write){
```

Comment:
Please remember to format.

Any reason you don't have a prettier extension installed in your editor?

Reply:
It's on my list to get it setup. Its more getting a list of rules that works for my style for when I'm not contributing to other projects that holding me back.

I'll get it up and running soon.